components/char_collection/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "icu-char-collection"
+description = "API for managing Unicode Language and Locale Identifiers"
+version = "0.0.1"
+authors = ["The ICU4X Project Developers"]
+edition = "2018"
+readme = "README.md"
+repository = "https://github.com/unicode-org/icu4x"
+license = "MIT/Apache-2.0"
```

Comment:
license-file here

Reply:
Rust throws a warning with both `license` and `license-file` in the `Cargo.toml`. Should we stick with just `license-file`?

Reply:
See https://github.com/unicode-org/icu4x/blob/master/components/locale/Cargo.toml for example of how to do it.